src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 4 failures in `PowerShell-CI-linux`
Test-Connection.Ping.Default parameter set is 'Ping'
```powershell
Expected the value to have type [System.Net.NetworkInformation.PingOptions] or any of its subtypes, but got $null with type $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 41
41:             $result[0].Options | Should -BeOfType "System.Net.NetworkInformation.PingOptions"
```
Test-Connection.MTUSizeDetect.MTUSizeDetect works
```powershell
A parameter cannot be found that matches parameter name 'DetectMtuSize'.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 212
212:             $result = Test-Connection $hostName -DetectMtuSize

```
Test-Connection.MTUSizeDetect.Quiet works
```powershell
A parameter cannot be found that matches parameter name 'DetectMtuSize'.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 221
221:             $result = Test-Connection $hostName -DetectMtuSize -Quiet

```
Test-Connection.TraceRoute.Quiet works
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 251
251:             $result | Should -BeTrue
```